thunder/__init__.py

```diff
@@ -569,6 +569,7 @@ def get_computation_and_inputs(*args, **kwargs):
             transform: Transform
             for transform in transforms:
                 thunder.core.utils.check_type(transform, Transform)
+
```

Comment:
Nitpicking: could you please remove the whitespace-only changes?

Reply:
sorry, I think this was due to precommit..